README.md

```diff
@@ -1 +1,44 @@
-# Awesome-Books
\ No newline at end of file
+# Awesome-Books
+
+> An app that allows the user to input the titles of books in his/her library along with the name of the author. The books can be removed from the library as they are read or discarded. This project is an exercise in Javascript DOM manipulation, objects and classes.
+
```

Comment:
**Changes Required**
You have a professional README, however;
- [ ] May we please an instructions on how the contributor/review can setup and run the app locally, this will allow for easier future collaboration.